completions/meson.build

```diff
@@ -0,0 +1,8 @@
+if get_option('fish-completions')
```

Comment:
You can use all the code from L29 - L38 here https://github.com/swaywm/swaylock/blob/f9ce3f193bc2f035790372b550547686075f4923/completions/meson.build#L29C1-L38C57 with the necessary edits. L28 can be put on L74 of our top meson.build, replacing the option string with 'install_shell_completions'.

L2 of the sway meson build, where the dependency is checked will go at the top of this file.

Reply:
you mean adding the

```
fish_comp = dependency('fish', required: false)
```

then 

```
if fish_comp.found()
	fish_install_dir = fish_comp.get_variable('completionsdir')
else
	fish_install_dir = datadir + '/fish/vendor_completions.d'
endif
```

I was unsure about its need and I didn't find it in other projects

Reply:
got it